tests/fixtures/config_sample.txt

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
File `.json`

Reply:
It is a "raw" file from Renault servers. It is badly formatted and I don't want black/lint/etc. to change it

Reply:
It make sens to keep it txt.